packages/babel-helper-wrap-function/src/index.ts

```diff
@@ -63,29 +66,69 @@ const buildDeclarationWrapper = template.statements(`
 function classOrObjectMethod(
   path: NodePath<t.ClassMethod | t.ClassPrivateMethod | t.ObjectMethod>,
   callId: t.Expression,
+  ignoreFunctionLength: boolean,
 ) {
   const node = path.node;
   const body = node.body;
 
+  let params: Array<t.Identifier | t.Pattern | t.RestElement> = [];
+
+  // Errors thrown during argument evaluation must reject the resulting promise
+  const shoudlForwardParams = node.params.some(p => p.type.includes("Pattern"));
```

Comment:
```suggestion
  const shoudlForwardParams = node.params.some(p => isPattern(p));
```
where `isPattern` can be imported from `@babel/types`.